pages/index.js

```diff
@@ -1,7 +1,9 @@
+import ServiceList from "../components/ServiceList/index";
+
 export default function HomePage() {
   return (
-    <div>
-      <h1>Hello from Next.js</h1>
-    </div>
+    <>
+      <ServiceList />
+    </>
```

Comment:
As you are returning only one `JSX element` here (ServiceList), you do not need the fragments 🙂 
```suggestion

      <ServiceList />

```